openedx/features/caliper_tracking/tests/expected/edx.drag_and_drop_v2.feedback.opened.json

```diff
@@ -0,0 +1,46 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "OpenedPopout",
```

Comment:
I believe a `Viewed` event is more appropriate here. `OpenedPopout` is specific to video popouts.

Reply:
done